### DISCUSSION TITLE
SharpMonoInjector.dll virus notification

### DISCUSSION THREAD

**Discussion by xEtarusx** [winstxnhdw/lc-hax#578] [General]
Hey,
tried to build/run the code.
Windows 11 reports the built "SharpMonoInjector.dll" from the submodule as a virus.

Is this known?

Virustotal scan of said file:
https://www.virustotal.com/gui/file/c548868f8ebc94aa99fefde6564d505db317d4ce20eb99e5035304ec2d940286

**Comment by winstxnhdw** [comment]
Hey, there's a bunch of discussions and issues on this topic already. It is also mentioned in the README. Please only open discussions or issues when you've exhausted your search options.
